pkgs/default.nix

```diff
@@ -28,18 +28,39 @@ rec {
     inherit pkgs;
   };
 
-  bundle-locked = pkgs.callPackage ./bundle-locked { inherit revstring; };
-
   bundle-image = pkgs.callPackage ./bundle-image {
     inherit bundle-locked revstring;
     inherit active-modules upgrade-maps;
   };
 
   bundle-image-tarball = pkgs.callPackage ./bundle-image-tarball { inherit bundle-image revstring; };
 
-  bundle-squashfs = pkgs.callPackage ./bundle-squashfs {
-    inherit bundle-locked revstring;
-    inherit active-modules upgrade-maps;
+  bundle-locked-fn = { moduleIds }: pkgs.callPackage ./bundle-locked {
+    inherit moduleIds;
+    inherit revstring;
+  };
+
+  bundle-locked = bundle-locked-fn {
+    moduleIds = null;
+  };
+
+  bundle-squashfs-fn = { moduleIds }: pkgs.callPackage ./bundle-squashfs {
+    bundle-locked = bundle-locked-fn {
+      inherit moduleIds;
+    };
+    inherit active-modules upgrade-maps revstring;
+  };
+
+  bundle-squashfs = bundle-squashfs-fn {
+    moduleIds = null;
+  };
+
+  basic-bundle-locked = bundle-locked-fn {
+    moduleIds = [ "python-3.10:v1-20230525-c48c43c" ];
+  };
+
+  basic-bundle-squashfs = bundle-squashfs-fn {
+    moduleIds = [ "python-3.10" "nodejs-18" ];
   };
```

Comment:
can we just generate an output for every module instead of this?

Reply:
@ryantm didn't like there being so many entries in `nix flake show`? Maybe we can nest in under another attrset like modules.

Reply:
hm, it's just weird to me having these outputs pre-defined and maybe arbitrarily changing in the future. maybe comment these out and leave a comment saying 'these are how you test the modules locally'?

Reply:
I'll document a good flow...

Reply:
This is what I have it mind: https://github.com/replit/goval/pull/10077